tests/conftest.py

```diff
@@ -71,6 +72,7 @@ def predictFastaiImage(self, input_data):
         def predictFastaiImages(self, original, compared):
             return all(original.data[0, 0] == compared.data[0, 0])
 
+
```

Comment:
Changes due to running the auto-formatting script. I can checkout these files if we don't want edits on them in this PR.

Reply:
No worries, just curious what version of `black` were you running"?